test/e2e/side_effects.go

```diff
@@ -205,9 +206,7 @@ func newEKSAPackagedBinaryForLocalBinary(tb testing.TB) eksaPackagedBinary {
 	}
 }
 
-func runTestManagementClusterUpgradeSideEffects(t *testing.T, provider framework.Provider, osFamily anywherev1.OSFamily, kubeVersion anywherev1.KubernetesVersion) {
-	latestRelease := latestMinorRelease(t)
-
+func runTestManagementClusterUpgradeSideEffects(t *testing.T, provider framework.Provider, latestRelease *releasev1.EksARelease, configFillers ...api.ClusterConfigFiller) {
```

Comment:
I would prefer if we didn't change the signature this way
the idea is that tests will only depend on provider, kube version and OS
So combinations can be easily parametrized

I'm ok if you need to add the extra fillers to make some stuff custom, but could we keep the rest as it was? that way we don't need to change the other provider tests either

Btw, there is an alternative. You could add the kindnetd filler to the default fillers added by the docker provider. That way it will be added to all docker tests by default

Reply:
i’d prefer the approach to add the extra fillers on then if that’s acceptable, and keep the rest of the signature as it was.

what we wanted to do was use kindnet for only the upgrade from latest minor release docker tests, instead of them all using kindnet by default